ldaptor/encoder.py

```diff
@@ -11,3 +13,23 @@ def to_bytes(value):
     if isinstance(value, six.text_type):
         return value.encode('utf-8')
     return bytes(value)
+
+
+def to_unicode(value):
```

Comment:
Do we need it as a public API? (sorry for missing this comment about `to_bytes`)

I think that we can keep it as a private API (name it with a _)

if it should only be used inside ldaptor it should be private (starting with underscore) and have doctrings :)

No need for dedicated tests (assuming that we have tests for the public API also covering this code)... but all public API should have tests.

Documentation is still nice to have on all code, public or private :)

Reply:
Hmm... Will it be okay if private ``_to_bytes`` and ``_to_unicode`` functions be imported in other ldaptor modules? I have not found such cases in the library yet. Anyway I added the docstrings, thanks for reminding about them :)

Reply:
Is ok to use private functions inside ldaptor.
The private is a sign to external ldaptor users that they should not use that code...or should use it at their own risk... but we will not warn them when that API is changed.

Reply:
So, ``encoder`` module was made private